docs/features/automation-rules.md

```diff
@@ -96,10 +96,13 @@ or send messages via a webhook URL. The Slack message body can be customized to
 You can also utilize [Slack incoming webhooks](https://slack.com/help/articles/115005265063-Incoming-webhooks-for-Slack)
 to send alerts.
 
-The `sendSlackNotification` function takes three arguments:
-* channel or webhook URL - String - destination for the message
-* message (optional) - String - if not set, Insights will construct a default message from the Action Item
-* isWebhook (optional) - Boolean - set to `true` if the first parameter is a webhook URL
+The `sendSlackNotification` function accepts these arguments:
+| Parameter              | Required? | Type    |                                                                            |   |   |   |   |   |   |
```

Comment:
you've got a bunch of extra columns here...

Reply:
Thanks @rbren just pushed a fix for the extra columns